tests/Feature/Repositories/OrderItemRepositoryTest.php

```diff
@@ -97,7 +97,7 @@ public function testUpdateUpdatesResource()
         ]);
 
         $this->assertTrue(
-            $this->repo->findById($item->id)->{$property} === $newValue
+            $this->repo->findById($item->id)->{$property} === (string) $newValue
```

Comment:
@mblarsen I'm going to remove these casts (there's another one in `OrderRepositoryTest`) that I added yesterday and instead fix this by casting the `orderable_id` as an `int` in the model, following your example.

Reply:
Ok. I actually didn’t look at the other tests as I wanted to confirm with you first.